index.html

```diff
@@ -50,6 +50,189 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
         </ul>
       </div>
     </section>
+    <section>
+      <div class="grid-container">
+        
+        <div class="card">
+          <div class="card-element">
+            <div class="img-container">
+              <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
+            </div>
+            <h2 class="card-title">Tonic</h2>
+            <div class="card-section">
+              <p class="para-one">CANOPY</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Back End Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
```

Comment:
- [ ] Please make sure that the button keeps the same size while the window resizes, to achieve this I suggest you only set a padding value for the button instead of a width value, in that way the button won't resize unless its text content is modified
![image](https://user-images.githubusercontent.com/52635378/164895590-6beba40d-1037-4860-9257-1b2191e358fe.png)

Reply:
Thank you for the suggestion. This design I style for specifically mobile width i.e 375px. I will consider a responsive version later.